### PR TITLE
Inter for `<Heading>` and theme.fonts.heading

### PR DESCRIPTION
**Pull request by fk**
- "soft-deprecate" `<Heading fontVariant="UI" />`
- adjust design-token docs

### CODE REVIEW

File: src/components/Breadcrumb/Breadcrumb.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const breadcrumbItemCss: ThemeCss = theme => ({
   display: `flex`,
   alignItems: `center`,
   color: theme.colors.purple[50],
-  fontFamily: theme.fonts.headingUI,
+  fontFamily: theme.fonts.heading,
```
Comment (fk) [inline src/components/Breadcrumb/Breadcrumb.tsx:34]: Only reference to `headingUI` from within `gatsby-interface` AFAICS.

File: src/components/Heading/Heading.tsx [hunk 2]
```diff
@@ -7,15 +7,15 @@ import { HeadingTone, HeadingVariant, HeadingFontVariant } from "./types"
 
 const baseCss: ThemeCss = theme => ({
   margin: 0,
-  lineHeight: theme.lineHeights.heading, // Ask Flo about this
+  lineHeight: theme.lineHeights.heading,
```
Comment (fk) [inline src/components/Heading/Heading.tsx:10]: This is fine. 👋 🤗 @Elanhant <3 <3 <3

**Review by aghreed — APPROVED** [review]
LGTM!

**Comment by greglobinski** [comment]
replacement https://github.com/gatsby-inc/gatsby-interface/pull/510

**Pull request closed without merging**